config/deploy/kubernetes/kubernetes-all.yaml

```diff
@@ -3681,6 +3681,7 @@ rules:
       - watch
       - create
       - update
+      - delete
```

Comment:
If the auth token is outdated, wouldn't it suffice to just update the secret?
Then we wouldn't need this permission